mlrun/model_monitoring/db/stores/sqldb/sql_store.py

```diff
@@ -500,22 +514,32 @@ def _delete_application_result(
     ) -> None:
         self._init_application_results_table()
         criteria = self._get_filter_criteria(
-            table=self.ApplicationResultsTable,
+            table=self.application_results_table,
+            endpoint_id=endpoint_id,
+            application_name=application_name,
+        )
+        # Delete the table
```

Comment:
Are you sure it deletes the table? I think it deletes only the relevant entries. 
The same for line 533.

Reply:
Right, I'll update the comment.